source/creating_content_with_studio/configuring_blocks/add-chart.rst

```diff
@@ -0,0 +1,86 @@
+Adding a chart
+==============
+
+You can use charts to create a graphic representation of your data.
+
+Step 1: Select the block type
+-----------------------------
+
+You need to define the block type before you configure the content you want to add.
+
+1. From the preview pane, click the block you want to edit.
+2. From the left side pane, select **Chart** as the block type.
+3. Click **Configure**.
+
+If you change your mind, you can change the block type afterward.
+The block configuration will be reset after you change the block type.
+
+Step 2: Select the dataset
+--------------------------
+
+Select the source dataset to create a graphic representation of the data.
+
+.. admonition:: Note
+    :class: note
+ 
+    The dataset is only selected for the current block.
+    If you add some content to another block afterward, you will need to select an input dataset again.
+
+1. From the left side pane, select the desired dataset from the list.
+
+   A preview of the first 20 records opens in the preview pane.
+
+   <PLACEHOLDER_SCREENSHOT_DATASET_RECORDS_PREVIEW>
```

Comment:
I may be too cautious but, just in case, I leave this little comment to remind us that there are many placeholders to replace.